pkg/jsonutil/conversion.go

```diff
@@ -20,17 +15,16 @@ type ConversionOption int
 // ConversionOption constant values.
 const (
 	OptCompact ConversionOption = iota
+	// OptUnEscape - is no-op, and it will be removed.
 	OptUnEscape
 )
 
 // JSONUnmarshaler returns a jsonpb Unmarshaler configured to allow unknown fields,
 // i.e. not error out unmarshaling JSON that contains attributes not defined in proto.
 // This Unmarshaler must be used everywhere instead of direct calls to jsonpb.Unmarshal
 // and jsonpb.UnmarshalString.
-func JSONUnmarshaler() *jsonpb.Unmarshaler {
-	return &jsonpb.Unmarshaler{
-		AllowUnknownFields: true,
-	}
+func JSONUnmarshaler() *protojson.UnmarshalOptions {
```

Comment:
Maybe it can be a singleton

Reply:
Documented for post-migration improvements.